openmetadata-service/src/main/resources/json/data/eventsubscription/WorkflowEvents.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "WorkflowEventConsumer",
+  "fullyQualifiedName": "WorkflowEventConsumer",
+  "displayName": "Workflow Event Consumer",
+  "description": "Consumers EntityChange Events in order to trigger Workflows, if they exist.",
+  "alertType": "GovernanceWorkflowChangeEvent",
+  "filteringRules":{
+    "resources": ["all"]
+  },
+  "destinations": [
+    {
+      "id": "fc9e7a84-5dbd-4e63-8b78-6c3a7bf04a60",
```

Comment:
what's this ID?

Reply:
Literally just a randomly generated ID that is used for this seed data